src/supermarket/spec/controllers/api/v1/cookbook_versions_controller_spec.rb

```diff
@@ -186,4 +186,81 @@
       end
     end
   end
+
+  describe '#collaborators_evaluation' do
+    let(:cookbook) { create(:cookbook) }
+    let(:version) { create(:cookbook_version, cookbook: cookbook) }
+
+    context 'the request is authorized' do
+      context 'the required params are provided' do
+        it 'returns a 200' do
+          post(
+            :collaborators_evaluation,
+            cookbook_name: cookbook.name,
+            collaborator_failure: false,
+            collaborator_feedback: 'This cookbook does not have sufficient collaborators.',
+            fieri_key: 'YOUR_FIERI_KEY',
+            format: :json
+          )
+          expect(response.status.to_i).to eql(200)
+        end
+
+        it "updates the cookbook version's collaborator attributes" do
+          post(
+            :collaborators_evaluation,
+            cookbook_name: cookbook.name,
+            cookbook_version: version.to_param,
+            collaborator_failure: false,
+            collaborator_feedback: 'This cookbook does not have sufficient collaborators.',
+            fieri_key: 'YOUR_FIERI_KEY',
+            format: :json
+          )
+
+          expect(version.reload.collaborator_failure).to eql(false)
+          expect(version.reload.collaborator_feedback).to eql('This cookbook does not have sufficient collaborators.')
```

Comment:
All of the times we use 'sufficient' language (or whatever language we want) should be in the internationalization section.